handlers/handlers_test.go

```diff
@@ -32,12 +32,12 @@ func TestSuccessfulVODUploadHandler(t *testing.T) {
 
 	catalystApiHandlers := CatalystAPIHandlersCollection{MistClient: StubMistClient{}, StreamCache: make(map[string]StreamInfo)}
 	var jsonData = `{
-		"url": "http://localhost/input",
+		"url": "http://0.0.0.0/input",
 		"callback_url": "CALLBACK_URL",
 		"output_locations": [
 			{
 				"type": "object_store",
-				"url": "memory://localhost/output",
+				"url": "memory://0.0.0.0/output",
```

Comment:
I think this host is ignored, but feels very weird to have memory://0.0.0.0. Might as well keep it localhost (and enforcing on parsing perhaps?)